cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/messages/CassandraStoredMessageFilter.java

```diff
@@ -49,7 +49,8 @@ public class CassandraStoredMessageFilter implements CassandraFilter<MessageBatc
 	private final FilterForLess<Instant> messageTimeTo;
 	private final FilterForAny<Long> sequence;
 
-	private final Integer limit;
+	/** limit must be strictly positive ( limit greater than 0 ) */
```

Comment:
Let's check this condition in constructor

Reply:
I had to revert this change. checking in constructor doesn't fit well, because there are cases when limit is 0 and it means that it simply is not applied.